tests/scancode/test_ignore_files.py

```diff
@@ -74,3 +81,45 @@ def test_ignore_glob_file(self):
             {}
         )
         assert is_ignored(*test)
+
+    def test_resource_paths_1(self):
+        test = (
+            test_dir,
+            {'*.doc': 'test ignore'}
+        )
+        expected = [
+            'user',
+            'user/src',
+            'user/src/test',
+            'user/src/test/sample.txt'
+        ]
+        for abs_path, rel_path in resource_paths(*test):
```

Comment:
Why using tuple unpacking rather than explicit named args? Also why using a global test_dir? There is no reason that each unit tests would be dependent on a global and dependent on each other.